cmd/influx/telegrafs.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/influxdata/influx-cli/v2/clients/telegrafs"
+	"github.com/influxdata/influx-cli/v2/pkg/cli/middleware"
+	"github.com/urfave/cli/v2"
+)
+
+func newTelegrafsCommand() *cli.Command {
+	var params telegrafs.ListParams
+	flags := append(commonFlags(), getOrgFlags(&params.OrgParams)...)
+	flags = append(flags, &cli.StringFlag{
+		Name:        "id",
+		Usage:       "Telegraf configuration ID to retrieve",
+		Aliases:     []string{"i"},
+		Destination: &params.Id,
+	})
+	return &cli.Command{
+		Name:  "telegrafs",
+		Usage: "List Telegraf configuration(s). Subcommands manage Telegraf configurations.",
```

Comment:
The old CLI has longer usage/example strings that I think we should carry forward. You can add `Description` fields with raw/multiline strings for that. For example, here's how `influx config` does it: https://github.com/influxdata/influx-cli/blob/9747d05ae1d3b726d2935ae8230724e58eba6442/cmd/influx/config.go#L231-L248